docs/kubernetes-build-win-binaries.md

```diff
@@ -4,18 +4,18 @@ The following instructions show how to deploy the Windows Kubernetes Binaries an
 
 ### Prerequisites
 * Azure Storage Account and Azure Storage Container to store Windows binaries
-* Access to [winnat.sys](https://blogs.technet.microsoft.com/virtualization/2016/05/25/windows-nat-winnat-capabilities-and-limitations/) stored in a storage container. (WinNAT) is used to provide required NAT networking functionality for Windows containers that will be included in a future Windows image update.
+* Access to [wincni.exe] (https://github.com/Microsoft/SDN/tree/master/Kubernetes/windows/cni) stored in a storage container. Windows CNI is a plugin that supports the Container Network Interface (CNI) network model and interfaces with the Windows Host Networking Service (HNS) to configure host networking and policy.
 * Docker installed and running. MacOS users using Docker for Mac must have at [least 3GB of memory allocated to Docker](https://github.com/kubernetes/kubernetes/tree/master/build/#requirements) or building will likely fail.
 
 ## Background
-Microsoft maintains a fork of the Kubernetes project at https://github.com/Azure/kubernetes which includes patches not yet included in upstream Kubernetes; these are needed for Windows containers to function.
+Microsoft maintains a fork of the Kubernetes project at https://github.com/Azure/kubernetes which includes patches not yet included in upstream Kubernetes for release 1.7 and 1.8; these are needed for Windows containers to function. *From release 1.9, all Windows features are in upstream and Windows binaries no longer needs to be built from Azure fork.*
```

Comment:
This is great. Can we move the background to the top as to make it front and center? Ideally most folks will see that from v1.9+ there's no need to do anything special for Windows nodes and hackers can continue reading if they want to build their own.